packages/mint-components/src/components/tax-and-cash/sqm-tax-document-submitted/sqm-tax-document-submitted.feature

```diff
@@ -0,0 +1,81 @@
+@owner:andy @author:andy
+Feature: Tax Document Submitted View
+  View that informs the user of their Tax Document submission status.
+
+  Background: A user has submitted their Tax Document
+    Given a user is on the Tax Document Submitted view
+
+  @motivating
+  Scenario: View Tax Document Submitted
+    Given the user has submitted a tax document
+    When they view Document Submitted
+    Then they see the status of their Tax Document and Banking Information
+
+  @motivating
+  Scenario: View Tax Document Submitted is loading
+    Given the user has submitted a tax document
+    When they view Document Submitted
+    Then they see a skeleton loaders until the end users Tax Document/Banking information data is available
+
+  @minutia @ui
+  Scenario Outline: The Tax Form header is displayed to the end users
+    Given the user has submitted a W9/W8-BEN/W8-BEN-E Tax form
+    Then the Tax Form header displays "<documentType> Tax Form"
+
+    Examples: 
+      | documentType |
+      | W9           |
+      | W8-BEN       |
+      | W8-BEN-E     |
+
+  @minutia @ui
+  Scenario Outline: The date submitted and status of a Tax Form is displayed to the user
+    Given the Tax Document is currently <status>
+    When they view the Tax Document Submitted
+    Then they see a badge with <status> text and a respective <badgeVariant>
+    Then they see a message indicating "<taxStatusMessage> on <dateSubmitted>"
+
+    Examples: 
+      | status       | badgeVariant | taxStatusMessage                                                          | dateSubmitted |
+      | ACTIVE       | success      | Submitted on                                                              | Jan 17, 2024  |
+      | NOT_VERIFIED | neutral      | Awaiting Review. Submitted on                                             | Jan 17, 2024  |
+      | NOT_ACTIVE   | danger       | Ensure your information matches your profile and resubmit a new document. | Jan 17, 2024  |
+
+  @minutia @ui
+  Scenario: Status badge and text does not appear if participant is not required to submit tax form
+    When a tax document is not required
+    Then tax document header does not include a document type
+    And the description text under the header displays "Tax documents are only required if you are based in the US. If your country of residence has changed, please contact Support"
+    And the "Submit new document" is not available
+
+  @minutia
+  Scenario Outline: Indirect Tax section shows details if participant is registered for indirect tax
```

Comment:
@Locrian24 @AndiLuo what does this look like now that the businessEntity stuff has changed a bit. A participant doesn't really have the idea of being a business entity anymore

Reply:
I think it stays the same granted they still select "individual participant" or "I represent a company" in step3.
- If they not registered in step 2 or select individual participant step 3, the Indirect tax details will show the not registered state
- Also, if they are registered in step 2 but select individual participant in step 3, it will still show the not registered state.
- The only time it will show Indirect Tax Details is if they are registered in step2 and select "I represent a company" in step 3
@DerekSiemens

Reply:
The issue here is that we are not storing the value of what they selected in step 3 (business entity or partner)

Reply:
I think what it boils down to now is that either they have indirect tax data and we show that or they don't and they get the not registered state. We can't really use the data from step 3 in our logic here

Reply:
Might be more of a hooks thing for @Locrian24 but the specs will have to change

Reply:
Ah I see, sorry I shall modify the code and specs to reflect that:
- If they select not registered in step2, it will always show the not registered state in step 4
- And whatever they select in step3 does not affect how the Indirect Tax Details render in step 4